modules/Kconfig.tfm

```
@@ -94,6 +94,30 @@ config TFM_BL2_FALSE
 
 endchoice
 
+config TFM_PARTITION_PROTECTED_STORAGE
+	bool "Enable secure partition 'Protected Storage'"
+	default y
```

Could you add some help text, describing what exactly this setting does, and that the reader can find additional info in Trusted Firmware M itself.

Maybe here would be a good place to state that dependencies between partitions are handled in TFM build system, and not Kconfig, as we have an exception here.

This goes for all the `config`s in this PR.

```suggestion
	help
	  This is what I do, and here you can find more info...
```

---

Check now, not sure exactly what to write.